build/Makefile

```diff
@@ -13,12 +13,12 @@
 
 # Define core directory below which everything resides. This is the
 # parent directory of the 'build' directory
-F_MASTER =
+F_MASTER = /home/wayandn/summa
```

Comment:
@NicWayand - please remove your own updates from the Makefile when it you make the PR. Simply remove this commit from the branch (or just revert your Makefile). If you do that on the same branch as you have used before: NicWayand:addDenFunc, then this PR will update automatically.